scripts/generate_wallets.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+cd src/threefold_token/token_overview/special_wallets/
+
+curl -L -X GET 'https://horizon.stellar.org/accounts/GA7OPN4A3JNHLPHPEWM4PJDOYYDYNZOM7ES6YL3O7NC3PRY3V3UX6ANM' \
```

Comment:
Let's suppress the output on these (some flag or redirect it to `/dev/null`), so it doesn't spam the terminal.

Reply:
Good idea. Took your suggestion of -sL instead of -L -X GET